libraries/AC_Avoidance/AP_OADatabase.cpp

```diff
@@ -440,14 +440,18 @@ void AP_OADatabase::send_adsb_vehicle(mavlink_channel_t chan, uint16_t interval_
         }
 
         // convert object's position as an offset from EKF origin to Location
-        const Location item_loc(Vector3f{_database.items[idx].pos.x * 100.0f, _database.items[idx].pos.y * 100.0f, _database.items[idx].pos.z * 100.0f}, Location::AltFrame::ABOVE_ORIGIN);
+        const AbsAltLocation item_loc{Vector3f{
```

Comment:
This doesn't seem right, the locations in the OADatabase are expected to be ABOVE_ORIGIN, but now they will be AMSL, won't that cause the OA code to break?